packages/vue/src/components/organisms/SfTabs/SfTabs.vue

```diff
@@ -35,6 +48,21 @@ export default {
       }
     },
   },
+  provide: function () {
+    const contentShow = {};
```

Comment:
@AdamPawlinski what do you think about the change it name to`tabConfig`

Reply:
I change that.